index.html

```diff
@@ -20,6 +20,32 @@
         </style>
     </head>
     <body>
+        <article>
+            <h2>Cosmogony metrics</h2>
+            <p>You can check below the number of zones by country for each type of zone, and compare this number with the actual number of zones in the real world.
```

Comment:
should we had something to tell how the found the values?  a link to our readme or mention wiikidata ? (but a mention to our readme might be enough)

Reply:
done